.eslintrc.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     "import"
   ],
   "parserOptions": {
-    "sourceType": "script"
+    "sourceType": "module"
```

Comment:
I'm fairly certain that this line can just be removed as module is the default for airbnb.